scanner/django.go

```diff
@@ -41,6 +44,69 @@ func configureDjango(sourceDir string, config *ScannerConfig) (*SourceInfo, erro
 
 	vars := make(map[string]interface{})
 
+	var pythonSupported string = "3.7" // https://devguide.python.org/versions/#supported-versions
+	var pythonVersion string = "3.10" // Keep the default version "3.10" (hardcoded on the Dockerfile)
+	var pythonVersionOutput string = "Python 3.10.0"
+	var pythonVersionFound bool = false
+
+	cmd := exec.Command("python3", "-V")
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		cmd := exec.Command("python", "-V")
+		out, err := cmd.CombinedOutput()
+		if err == nil {
+		    pythonVersionOutput = string(out)
+		}
+	} else {
+	    pythonVersionOutput = string(out)
+	}
+
+    re := regexp.MustCompile(`Python ([0-9]+\.[0-9]+)\.[0-9]`)
```

Comment:
How about extracting this part as a separate function? It would make testing the logic easier.

Reply:
Sure, I'll do that! Thanks for the suggestion!